homeassistant/components/tplink/strings.json

```diff
@@ -42,16 +42,36 @@
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]"
         }
+      },
+      "reconfigure": {
+        "title": "Reconfigure TPLink entry",
+        "description": "Will update configuration for device {mac}",
```

Comment:
Many other integrations do seemingly define only `description` with something like `Update your configuration for {name}.`. Could we skip the title and use name instead of mac here, too?

Reply:
I think it's important to use mac because the reconfigure flow allows a user to change the IP address in the event of a device changing it's ip (and discovery not working to automatically update it). In this instance the name would be incorrect as well as the IP so I think mac is only true identifier.

Reply:
Ok, that's fine, but how about the suggestion to change the wording a bit?

Reply:
That's done: `Update your configuration for device {mac}`